komga-webui/src/services/komga-books.service.ts

```diff
@@ -95,4 +96,18 @@ export default class KomgaBooksService {
       throw new Error(msg)
     }
   }
+
+  getSeries (komgaSeries: KomgaSeriesService, book: BookDto): Promise<SeriesDto> {
+    return komgaSeries.fromBook(book)
+  }
+
+  async getBookTitle (komgaSeries: KomgaSeriesService, book: BookDto): Promise<string> {
```

Comment:
services should do the binding with the server API, not much more. The logic to build a book title should either be located directly in the component using it, or in another file if it's used more than once, or server side if it makes sense

Reply:
Yes, maybe it would be best to determine the full title server side, that way the series doesn't have to be determined at all for the book.